save-backend/src/main/kotlin/com/saveourtool/save/backend/utils/ReactorUtils.kt

```diff
@@ -94,14 +94,20 @@ fun <T> justOrNotFound(data: Optional<T>, message: String? = null) = Mono.justOr
     message
 }
 
+/**
+ * @param supplier it returns nullable [T]
+ * @return [Mono] from result or empty
+ */
+fun <T : Any> lazyToMono(supplier: () -> T?): Mono<T> = Mono.just(UInt)
```

Comment:
What is the difference with `Mono.fromCallable`?

Reply:
Unit, not Uint :D

Reply:
it checks on nullable. `Mono.fromCallable` expects only not null value. 
This method creates a mono or Mono.empty

Reply:
Quoting from docs,

> If the Callable resolves to null, the resulting Mono completes empty.

I believe, `Mono.fromCallable` extects a non-nullable *callable*, which *can* emit `null`. Or am I misreading it?

Reply:
yep, you are right. confused by type resoling in idea in next map function (IDEA detects it as nullable).
actually, kotlin extension already has similar override. will re-use